app/Exports/UsersExport.php

```diff
@@ -7,20 +7,32 @@
 use App\Exports\UsersSheets\StatusesExport;
 use App\Exports\UsersSheets\StudentsCouncilFeedback;
 use App\Models\Role;
+use App\Models\User;
+use Illuminate\Database\Eloquent\Builder;
 use Maatwebsite\Excel\Concerns\WithDefaultStyles;
 use Maatwebsite\Excel\Concerns\WithMultipleSheets;
 use PhpOffice\PhpSpreadsheet\Style\Style;
 
 class UsersExport implements WithMultipleSheets, WithDefaultStyles
 {
+    private Builder|User $includedUsersQuery;
+
+    public function __construct(Builder|User $includedUsersQuery)
+    {
+        $this->includedUsersQuery = $includedUsersQuery;
+    }
+
     public function sheets(): array
     {
-        $sheets = [new CollegistsExport(), new StatusesExport()];
+        $sheets = [
+            new CollegistsExport($this->includedUsersQuery->clone()),
```

Comment:
why do you need `clone`?

Reply:
Different classes use the query passed to them in differing ways. By cloning, we ensure that modifications in one export class don't modify the behavior of the consequent exports.